server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMessagesCreationProcessorTest.java

```diff
@@ -179,6 +183,7 @@ protected MessageWithId createMessageInOutboxAndSend(ValueWithId.CreationMessage
         assertThat(result.getNotCreated()).isEmpty();
     }
 
+    @Ignore("JAMES-1716 : should report an error")
```

Comment:
ok understood: as your test checking that storing into this mailbox is before checking if the mailbox exist, you get an other error than the expected one